settings/default_docker/network.lua

```diff
@@ -0,0 +1,34 @@
+-----------------------------------
```

Comment:
There has to be a better way of providing defaults to docker that doesn't involve polluting the build for all other non-docker systems?

Reply:
Given that we're on the fence about dumping any in-repo docker files anyway, this doesn't sit well